indico/modules/events/contributions/controllers/display.py

```diff
@@ -168,7 +169,7 @@ class RHContributionJSON(RHContributionDisplayBase):
 
     def _process(self):
         from indico.modules.events.contributions.schemas import FullContributionSchema
-        can_manage = self.contrib.can_manage(session.user)
+        can_manage = self.contrib.can_manage(session.user, permission='contributions')
```

Comment:
That change wasn't correct, it's a management check on the contribution so there we need to check for full management access. Anyway, I reverted this in 986469c4d2.

```
ValueError: permission 'contributions' is not valid for 'Contribution' objects
```